design/general-progress-monitoring.md

```diff
@@ -0,0 +1,499 @@
+# Backup and Restore Item Action Progress Monitoring
```

Comment:
Thoughts on calling this something more user friendly?

A term that most users regardless of plugin development experience will understand is the term plugin.
The term item action may not be familiar to everyone.
```suggestion
# Plugin Progress Monitoring
```

Reply:
updating